types/local-ipv4-address/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for local-ipv4-address 0.0
+// Project: https://github.com/Aldaviva/local-ipv4-address#readme
+// Definitions by: Josh Bacon <https://github.com/CrispyBacon1999>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function main(): Promise<string>;
```

Comment:
Nit: name this `localIpV4Address` instead of `main` to help TypeScript offer correct auto-imports for this module.